leafsmart-front-end/src/App.js

```diff
@@ -4,6 +4,7 @@ import Events from "./components/Events"
 import "./App.scss";
 import CityFacts from "./components/CityFacts";
 import QoLData from "./components/QoLData";
+import Search from "./components/Search";
 import './styles/main.css';
 
 const App = () => {
```

Comment:
Are we ready to let go of the commented out code?

Reply:
I am ok with removing it as we would be using techincally something different for log in log out right?

Reply:
Yep! I think log in/out would use a similar kind of idea to "talk" to our api, which all just mirrors the API calls we're making to third(and fourth... and fifth) party endpoints.

Reply:
fine by me!